pkgs/applications/audio/friture/default.nix

```diff
@@ -28,15 +28,6 @@ in py.buildPythonApplication rec {
     rtmixer
   ];
 
-  patches = [
-    # Backported fix that resolves an issue with setuptools packaging
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
You're right, I totally forgot about that. Fixed it now